docs/prover/cli/options.md

```diff
@@ -671,8 +671,21 @@ Prover.  `--settings` should be followed by a comma-separated list of options.
 This list is incomplete.
 ```
 
+(-optimisticFallback)=
+#### `--settings -optimisticFallback=true`
+
+This option determines whether to optimistically assume unresolved external
+calls can make arbitrary changes to all states. By default unresolved external
+calls will havoc all states. When `-optimisticFallback` is enabled, only the
+states of contracts that are external to the calling contract (not necessarily
+`currentContract`) will be havoced.
+
+Enabling this option is safe when checking invariants and parametric rules are
+used and therefore all of the functions of the current contract are checked
+independently.
```

Comment:
If I understand correctly, this doesn't seem true... even though I have checked that a method satisfies a property doesn't mean that it doesn't change the state of the contract.